cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -4617,28 +4796,76 @@ counttype = number of events
-action.cyences_notable_event_action.attacker_drilldown = `cs_gsuite` sourcetype="gws:reports:admin" "event.name" IN ("CREATE_USER", "DELETE_USER") | rename "actor.email" as admin | search admin=$row.admin|s$ | stats count by user
+action.cyences_notable_event_action.param.filter_macro_name = cs_gws_successful_login_from_unusual_country_filter
+action.cyences_notable_event_action.contributing_events = `cs_gsuite_login_events` "event.name"="login_success" `cs_public_ips(ipAddress)` | iplocation ipAddress
+action.cyences_notable_event_action.system_compromised_search = | stats count by user
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_gsuite_login_events` "event.name"="login_success" `cs_public_ips(ipAddress)` user=$row.user$ | iplocation ipAddress
 action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Google Workspace
+
+
+[GWS - Suspicious Login Activity by User]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 9,39 * * * *
+description = This alert will show the suspicious login activity. \
+\
+Data Collection - Google login activity reports data from Splunk Add-on for Google Workspace (https://splunkbase.splunk.com/app/5556)
+dispatch.earliest_time = -2h@h
+dispatch.latest_time = +2h@h
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_gsuite_login_events` "event.name"="suspicious_login" _index_earliest=-31m@m _index_latest=-1m@m \
+| stats count, values(ipAddress) as ipAddress, max(_time) as Last_Success_Login by user \
+| iplocation ipAddress \
+| fields - lat lon \
+| sort - count \
+| `cs_human_readable_time_format(Last_Success_Login)` \
+| eval cyences_severity = "low" \
+| `cs_gws_suspicious_login_by_user_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_gws_suspicious_login_by_user_filter
+action.cyences_notable_event_action.contributing_events = `cs_gsuite_login_events` "event.name"="suspicious_login"
+action.cyences_notable_event_action.system_compromised_search = | stats count by user
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_gsuite_login_events` "event.name"="suspicious_login" user=$row.user$
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Google Workspace
+
+
+# TO DO - Add the DLP alert (regarding sharing the sensitive info) if it generates from the gws:reports:alert sourcetype
+# TO DO - Add the generic google alert from gws:reports:alert sourcetype
+# TO DO - Add the alerts related to the email logs
+# TO DO - Add the daily alert for the list of login failure happens in a day. (currently, data is not present)
```

Comment:
No login data? We definitely need to collect login data.

Reply:
Yes, we have login data for some event.name but we don't have login events with event.name=login_failure
I tried to login to the GWS admin account by entering invalid password but event is not generated. I asked the same to Rahaf.